lib/go-atscfg/chkconfig.go

```diff
@@ -26,17 +26,35 @@ import (
 	"github.com/apache/trafficcontrol/lib/go-tc"
 )
 
+// ChkconfigFileName is the name of the ckconfig configuration file.
 const ChkconfigFileName = `chkconfig`
+
+// ChkconfigParamConfigFile is the ConfigFile value of Parameters that affect
+// the generation of the chkconfig configuration file.
 const ChkconfigParamConfigFile = `chkconfig`
+
+// ContentTypeChkconfig is the MIME content type of the contents of the
+// chkconfig configuration file.
+//
+// TODO: The GoDoc for MakeChkconfig says "This is a JSON object, and should be
```

Comment:
I believe at one point the old ort.pl read the JSON generated by the config generation and executed Linux chkconfig commands based on it.

We now get that data from Traffic Ops, via the `t3c-request` tool, and generate it there, rather than `t3c-generate` (which generates from `lib/go-atscfg`).

Which is to say, it was sort of correct in both places, because the file was never written as JSON. But it's also kind of fuzzy. And I don't think it's used anymore.

But please don't delete it without verifying. I'm not 100% sure, someone will need to verify.

Reply:
I won't be deleting anything in this PR. Except possibly comments.